### PR TITLE
Edits to Networking 102

### PR DESCRIPTION
**Pull request by klevenstein**

### CODE REVIEW

File: labs/networking-102-the-topologies/1.md [hunk 1]
```diff
@@ -1,50 +1,56 @@
 
-# Networking 102 Topologies and Models #
-
-In this Learning Lab, you learn about various network topologies and cables along with the OSI and TCP/IP Suite network models.
+# Networking 102 Topologies and Models
 
+In this Learning Lab, you learn about network topologies, network cables, and the OSI and TCP/IP network models.
 
 ## Objective ##
```
Comment (annegentle) [inline labs/networking-102-the-topologies/1.md]: Can remove the extra ending `##` here also.

File: labs/networking-102-the-topologies/1.md [hunk 2]
```diff
@@ -1,50 +1,56 @@
 
-# Networking 102 Topologies and Models #
-
-In this Learning Lab, you learn about various network topologies and cables along with the OSI and TCP/IP Suite network models.
+# Networking 102 Topologies and Models
```
Comment (annegentle) [inline labs/networking-102-the-topologies/1.md]: I wonder if we should generally remove 101, 102 everywhere? What do you think?

Also use sentence case for headings, so "Networking topologies and models" may suffice here.

File: labs/networking-102-the-topologies/1.md [hunk 3]
```diff
@@ -1,50 +1,56 @@
...
 * Study the networking OSI and TCP/IP models
 
+This Learning Lab is based on Internet Protocol version 4 (IPv4). IPv6 and wireless networking are not covered.
 
 ## Prerequisites
 
-* You have some knowledge about networks or have completed the lab Networking 101.
-* Your focus in this lesson will be on wired networks which for your purposes is what you need to learn. Wireless networks have other network components not covered here.
-* This topic is based on IPv4 protocol; otherwise known as Internet Protocol version 4.
+To complete this lab, you should have completed the Networking 101 Learning Lab, or have equivalent high-level knowledge of networking basics, routers and switches, and SDN and REST.
```
Comment (annegentle) [inline labs/networking-102-the-topologies/1.md]: Hm, if removing 101 from titles, then also would have to scrub for cross references.

File: labs/networking-102-the-topologies/2.md [hunk 4]
```diff
@@ -1,23 +1,32 @@
...
 ![](/posts/files/networking-102-the-topologies/assets/images/twisted-pair.png)
 *Unshielded and Shielded Twisted Pair Wires*
 
-### What are the different network cables categories?
+## Network cable categories
 
 Network devices are connected by network cables. However, the throughput, allowed distance, and other characteristics of these cables vary, depending on the specific rating of the cable.
 
-![](/posts/files/networking-102-the-topologies/assets/images/cable-category.png)
 *Ethernet Cable Categories*
```
Comment (annegentle) [inline labs/networking-102-the-topologies/2.md]: Was this image particularly non-useful?

File: labs/networking-102-the-topologies/2.md [hunk 5]
```diff
@@ -1,23 +1,32 @@
...
 
-### What are straight and crossover network cables?
-To determine if you have a straight or crossover network cable, you can typically read the description written on the cable. However, a faster and more accurate way would be to take both ends of the cable in your hands and put the RJ45 male ends side-by-side. If the colored wires are in the same order on both RJ45's, then you have a straight cable. If the colored wires are in a different order, you have a crossover cable.
+## Straight and crossover network cables?
+
+A network cable usually have a description printed on them that indicates whether it is straight or crossover. You can also hold the RJ45 male ends of the cable side-by-side. If the colored wires are in the same order on both RJ45s, it is a straight cable. If the colored wires are in a different order, it is a crossover cable.
 
 ![](/posts/files/networking-102-the-topologies/assets/images/straight.png)
 *Straight Ethernet Cable*
```
Comment (annegentle) [inline labs/networking-102-the-topologies/2.md]: These image references can have /posts/files/lab-id/ removed while you're editing.

File: labs/networking-102-the-topologies/2.md [hunk 6]
```diff
@@ -26,19 +35,26 @@ To determine if you have a straight or crossover network cable, you can typicall
...
 
-![](/posts/files/networking-102-the-topologies/assets/images/cross-straight-cables.png)
-*Devices and Cables*
+This table reveals certain patterns.
 
-From the table shown one might see that there are certain patterns.
-  * Use a crossover cable to connect any device that connects to the same device type. For example, use a crossover cable to connect a router to another router, a switch to another switch or a computer to another computer. There are two exceptions to this rule: Use a crossover cable to connect a hub to a switch and connect a router to a computer, which is uncommon.
-  * Use a straight cable when connecting different devices. For example, when connecting a switch to a router, a switch to a computer, a hub to a router or a hub to a computer, use a straight cable.
+* Use a crossover cable to connect identical device types, except in the case of hub-switch and router-workstation connections. Router-workstation connections are uncommon.
+* Use a straight cable to connect different device types.
```
Comment (annegentle) [inline labs/networking-102-the-topologies/2.md]: Geez, good re-write. Much easier to read! 💯

File: labs/networking-102-the-topologies/2.md [hunk 7]
```diff
@@ -26,19 +35,26 @@ To determine if you have a straight or crossover network cable, you can typicall
...
 
+## Something to Consider
```
Comment (annegentle) [inline labs/networking-102-the-topologies/2.md]: Something to consider -- Sentence case, I won't mark through out but keep an eye out. 👁

File: labs/networking-102-the-topologies/networking-102-the-topologies.json [hunk 8]
```diff
@@ -5,13 +5,17 @@
   "time": 15,
   "byod": true,
   "files": [
-    {"title": "1.md"},
-    {"title": "2.md"},
-    {"title": "3.md"},
-    {"title": "4.md"}
+    {"title": "Networking 102 Topologies and Models",
```
Comment (annegentle) [inline labs/networking-102-the-topologies/networking-102-the-topologies.json]: Would change the 102 here if we eliminate everywhere, thanks for getting the titles entered and matching!

File: labs/networking-102-the-topologies/4.md [hunk 9]
```diff
@@ -1,20 +1,23 @@
...
+| 3     | Transport      | Provides reliable or best-effort data delivery with optional error and flow control                                                                                                                                           | TCP, UDP                                                  | Segments     |                                    |
+| 2     | Internet       | Provides logical end-to-end network addressing and routing.                                                                                                                                                                   | IP                                                        | Packets      | Router                             |
+| 1     | Network Access | Encapsulates IP packets into frames for transmission. Maps IP addresses to physical hardware addresses (MAC addresses) and uses protocols for physical data transmission.                                                     | 802.3, 802.2, HDLC, FDDI, PPP, Frame Relay                | Frames, Bits | Switch, Bridge, Hub, Repeater, NIC |
 *TCP/IP Suite Functionality*
```
Comment (annegentle) [inline labs/networking-102-the-topologies/4.md]: Yes! Text tables over images every day of the week.

File: labs/networking-102-the-topologies/1.md (hunk 2 above)
Comment (klevenstein) [reply labs/networking-102-the-topologies/1.md]: Yeah, I like the idea of removing the course numbers. I'll do that in a pass after the existing 102 and 103 PRs are merged.

**Pull request merged**